### ISSUE TITLE
[BUG] AMD Work Order not populating on Signal Details page - WK21-134840

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#7981]
Frank G. 
AMD

Emailed
>I noticed a problem with a few work orders not populating on the signal page. WK21-134840 signal # 733 ( Southwest Pkwy at Travis Cook ) not populating on the signal page.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           

**Describe the bug**
Would go to the Signals Page (ex. 733) and expect to see the Work Order (WK21-13480), but it's not showing up. It's only showing the related Service Request record.

**Comment by dianamartin** [comment]
I looked into this issue and experienced the same issues that Frank was having. I couldn't find the work order under the table. 

- [x] I added the "key word search" on the Service Request and Work Order table

Sync'd with Frank at 1:45 pm and walked through it together. Tried to add a new Work Order table to the table to see if there was a difference and there was!

Existing WO table: _This table displays work_orders_signal records connected to the same location connected to this page's signal._

New WO table: _This table displays work_orders_signal records connected to this page's signal._
- [x] Configured the new work order table to match the fields from the old table
- [x] Double checked the details tables to make sure they matched up
